src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -3,45 +3,242 @@
 import java.util.List;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+
+    private static class Node<T> {
```

Comment:
Make `Node<T>` an inner class. It should be placed at the end of the class, not at the beginning.

Reply:
Done